### PR TITLE
Enable semantic-release

### PR DESCRIPTION
**Pull request by ggrossetie**
Alternatively, we could run the task `npm run semantic-release` (defined in `package.json`):

```
semantic-release pre && npm publish && semantic-release post
```

### CODE REVIEW

File: .github/workflows/semantic-release.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      - uses: actions/checkout@v2
+      - name: Set up Node 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Install dependencies
+        run: npm ci
+      - name: Lint and test
+        run: npm t
+      - name: Release
```
Comment (sapegin) [inline .github/workflows/semantic-release.yml:21]: I think we need to pass secrets here, not sure it works like this by default.

https://github.com/sapegin/mrm/blob/d2d81c1d8c42b236b7a42909b24e1dc9bba8cd10/packages/mrm-task-semantic-release/index.js#L9-L43

**Review by sapegin — COMMENTED** [review]
You could also run `npx mrm semantic-release` to add the workflow.

File: .github/workflows/semantic-release.yml (hunk 1 above)
Comment (ggrossetie) [reply .github/workflows/semantic-release.yml:21]: Yes, you're right.
I've added both `GITHUB_TOKEN` and `NPM_TOKEN` as environment variables (you will need to configure them as secrets on GitHub if not already done)

File: .github/workflows/semantic-release.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        run: npx semantic-release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (sapegin) [inline .github/workflows/semantic-release.yml]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```

**Pull request merged**